ruby_03-professional_rails_applications/building-a-vps.markdown

```diff
@@ -667,7 +683,7 @@ test:
 
 production:
   <<: *defaults
-  socket_url: "http://<YOUR_IP_HERE>:4200"
+  socket_url: "http://104.236.170.113:4200"
```

Comment:
can you change this back i had added it in order to avoid people getting confused and putting the 104... ip